docs/openshift/acm.md

```diff
@@ -0,0 +1,123 @@
+# Advanced Cluster Management (ACM) Setup
+
+This guide shows you how to set up Red Hat Advanced Cluster Management (ACM) and use the OpenShift MCP Server with multiple OpenShift clusters.
+
+## Overview
+
+Advanced Cluster Management (ACM) allows you to manage multiple OpenShift clusters from a single hub cluster. The OpenShift MCP Server allows interaction with all your managed clusters through a single MCP server instance.
+
+## Step 1: Install ACM on Hub Cluster
+
+The hub cluster is the central OpenShift cluster that manages other clusters (managed clusters). You need to install ACM on the dedicated hub cluster first.
+
+### Complete Installation
+
+Install all ACM components with a single command:
+
+```bash
+make acm-install
+```
+
+This installs:
+1. MultiCluster Engine (MCE)
+2. ACM Operator
+3. ACM Instance (MultiClusterHub CR)
+
+### Verify Installation
+
+Check ACM installation status:
+
+```bash
+make acm-status
+```
+
+Expected output:
+```
+==========================================
+ACM Installation Status
+==========================================
+
+Namespaces:
+multicluster-engine   Active   5m
+open-cluster-management   Active   5m
+
+Operators:
+NAME                                 DISPLAY                              VERSION   REPLACES   PHASE
+advanced-cluster-management.v2.14.0  Advanced Cluster Management         2.14.0               Succeeded
+
+MultiClusterHub:
+NAME               STATUS    AGE
+multiclusterhub    Running   5m
+
+ManagedClusters:
+NAME           HUB ACCEPTED   MANAGED CLUSTER URLS   JOINED   AVAILABLE   AGE
+local-cluster  true           https://...            True     True        5m
+```
+
+## Step 2: Import Managed Clusters
+
+Once ACM is installed on your hub cluster, you can import additional OpenShift clusters.
+
+### Import a Cluster
+
+```bash
+make acm-import-cluster \
+  CLUSTER_NAME=production-east \
+  MANAGED_KUBECONFIG=/path/to/production-east-kubeconfig
+```
+
+**Parameters**:
+- `CLUSTER_NAME`: Unique name for the managed cluster
```

Comment:
Does this have to match a cluster name in the kubeconfig ?

Reply:
nope